app/api/datastore/v2/searches.rb

```diff
@@ -9,6 +9,7 @@ class Searches < Base
           optional :search, type: JSON, desc: 'Search JSON.' do
             optional :application_id_in, type: Array
             optional :application_id_not_in, type: Array
+            optional :status, type: Array[String], values: LaaCrimeSchemas::Types::APPLICATION_STATUSES
```

Comment:
How does the type `Array[String]` differ from Array? I get it is an "array of strings" but find it interesting that DryTypes does a check into the Array itself - all the better if it does 👍